.travis.yml

```diff
@@ -5,9 +5,7 @@ cache: pip
 before_install:
   - pip install --upgrade pip
 install:
-  - pip install -e .[test] --process-dependency-links
-  - pip install codecov
-  - pip install flake8
+  - pip install -r dev-requirements.txt
```

Comment:
I think separately installing `codecov` is still useful. We only want it on Travis (to post coverage to codecov), so it doesn't really belong in `dev-requirements.txt`.